libbeat/docs/highlights-7.0.0.asciidoc

```diff
@@ -0,0 +1,103 @@
+[[release-highlights-7.0.0]]
+=== 7.0.0 release highlights
+++++
+<titleabbrev>7.0.0</titleabbrev>
+++++
+
+Each release of {beats} brings new features and product improvements. 
+Here are the highlights of the new features and enhancements in 7.0.0.
+
+Refer to the {beats} <<breaking-changes-7.0, Breaking Changes>> and <<release-notes, 
+Release Notes>> for a list of bug fixes and other changes.
+
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+
+[float]
+==== Elastic Common Schema (ECS)
+
+The {ecs-ref}/index.html[Elastic Common Schema], or ECS, is an open source
+specification that defines a common set of document fields for event data
```

Comment:
nit: open source specification sounds weird to me. Would make open standard sense here?

@webmat WDYT?

Reply:
@urso It's the wording we've been using in a bunch of places. I agree it sounds a bit weird.

However I would not change it to open standard: although we're very open to external contributions, ECS is not being run like an open standard.

Reply:
I'm leaving as-is to match the blog, but we can always revisit the decision later when we truly have an "open standard"